portfolio_optimization/cuFOLIO_portfolio_optimization/CVaR/03_advanced_topics.ipynb

```
@@ -100,7 +100,10 @@
     "# Enable this in case you are running this in google colab or such places where cuOpt is not yet installed\n",
     "#!pip uninstall -y cuda-python cuda-bindings cuda-core\n",
     "#!pip install --upgrade --extra-index-url=https://pypi.nvidia.com cuopt-cu12 nvidia-nvjitlink-cu12 rapids-logger==0.1.19\n",
-    "#!pip install --upgrade --extra-index-url=https://pypi.nvidia.com cuopt-cu13 nvidia-nvjitlink-cu13 rapids-logger==0.1.19"
+    "#!pip install --upgrade --extra-index-url=https://pypi.nvidia.com cuopt-cu13 nvidia-nvjitlink-cu13 rapids-logger==0.1.19\n",
+    "\n",
+    "# Install other dependencies (if not already installed)\n",
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)